### PR TITLE
Certification power board

### PR DESCRIPTION
**Pull request by jspteampaydock**

### CODE REVIEW

File: assets/js/frontend/form.js [hunk 1]
```diff
@@ -181,41 +179,17 @@ setTimeout(() => jQuery(function ($) {
...
-                }
-            }, 1000)
-        })
-    }
-
-    function reloadWidget() {
-        window.widgetPowerBoard.reload();
-        const paymentSourceToken = document.querySelector('[name="payment_source_token"]');
-        paymentSourceToken.value = null;
-        window.widgetReloaded = true
```
Comment (jpcostello) [inline assets/js/frontend/form.js:218]: @jspteampaydock These were fixes added. Please pull changes before creating a PR.

**Review by jpcostello — CHANGES_REQUESTED** [review]

File: assets/js/frontend/form.js (hunk 1 above)
Comment (jspteampaydock) [reply assets/js/frontend/form.js:218]: Ok, thanks , no problem

File: includes/Services/ProcessPayment/ApmProcessor.php [hunk 2]
```diff
@@ -158,9 +158,7 @@ protected function getAdditionalFields( $exclude = [] ): array {
 			return [];
 		}
 
-		if ( ! is_admin() ) {
-			WC()->cart->calculate_totals();
-		}
+		WC()->cart->calculate_totals();
```
Comment (FabianaRBarros) [inline includes/Services/ProcessPayment/ApmProcessor.php:161]: I think this was part of a fix from Andrii. @andryes can you please confirm?

Comment (FabianaRBarros) [reply includes/Services/ProcessPayment/ApmProcessor.php:161]: Same for all cases like this one, for example: `BankAccountProcessor`, `CardProcessor`, `BankAccountBlock`

File: includes/Services/ProcessPayment/CardProcessor.php [hunk 3]
```diff
@@ -676,10 +666,6 @@ public function createCustomer( $force = false ): void {
 			], $this->getAdditionalFields( 'amount' ) ),
 		], $this->getAdditionalFields( 'amount' ) );
 
-		if ( empty( $customerArgs['phone'] ) ) {
-			unset( $customerArgs['phone'] );
-		}
-
```
Comment (FabianaRBarros) [inline includes/Services/ProcessPayment/CardProcessor.php:682]: Please re-add this as this is part of a bug fix

File: resources/js/frontend/index.js [hunk 4]
```diff
@@ -22,7 +22,6 @@ const labels = {
     saveCardLabel: __('Save payment details', textDomain),
     selectTokenLabel: __('Saved payment details', textDomain),
     fillDataError: __('Please fill in the card data.', textDomain),
-    fillCCDataError: __('Please fill in the required credit card form fields', textDomain),
```
Comment (FabianaRBarros) [inline resources/js/frontend/index.js:25]: Please re-add this as this is part of a bug fix

**Review by FabianaRBarros — CHANGES_REQUESTED** [review]

File: includes/Services/ProcessPayment/CardProcessor.php [hunk 5]
```diff
@@ -579,10 +577,6 @@ private function customerCharge(): array {
 				],
 			], $this->getAdditionalFields( 'amount' ) );
 
-			if ( empty( $customerArgs['phone'] ) ) {
-				unset( $customerArgs['phone'] );
-			}
-
```
Comment (FabianaRBarros) [inline includes/Services/ProcessPayment/CardProcessor.php:585]: Please re-add this as this is part of a bug fix

File: includes/Services/ProcessPayment/CardProcessor.php [hunk 6]
```diff
@@ -638,10 +632,6 @@ private function customerCharge(): array {
 			],
 		];
 
-		if ( empty( $params['customer']['phone'] ) ) {
-			unset( $params['customer']['phone'] );
-		}
-
```
Comment (FabianaRBarros) [inline includes/Services/ProcessPayment/CardProcessor.php:644]: Please re-add this as this is part of a bug fix

**Review by FabianaRBarros — APPROVED** [review]

**Review by andryes — APPROVED** [review]

**Review by andryes — APPROVED** [review]

**Pull request merged**